### PR TITLE
toggleable blue links with command (.links) - please delete the PR by account titled deatrys

### PR DESCRIPTION
**Pull request by h6b45**

### CODE REVIEW

File: profanity.rb [hunk 1]
```diff
@@ -1640,6 +1647,19 @@ def get_color_pair_id(fg_code, bg_code)
             elsif RbConfig::CONFIG['host_os'] =~ /linux|bsd/
               system "xdg-open #{url} >/dev/null 2>&1 &"
             end
+					elsif xml =~ /^<a/
+						if blue_links						
+							h = { :start => start_pos }
+							h[:fg] = '6060ff'
+							h[:bg] = nil
```
Comment (mrhoribu) [inline profanity.rb:1654]: Should be set to user-definable via xml template:
```ruby
							h[:fg] = PRESET['links'][0]
							h[:bg] = PRESET['links'][1]
```
Then just need to edit the default xml to have the preset defined:
https://github.com/elanthia-online/ProfanityFE/blob/master/templates/default.xml
```xml
  <preset id='links' fg='6060ff'/>
```

**Comment by h6b45** [comment]
Committed with <presets> in Layout XML templates, and calls the presets for the coloring.
It uses 5555ff for the default blue for them.
Still does not save across sessions whether they are toggled on or not.

**Comment by h6b45** [comment]
presets working, with background of '000000' not force black but rather transparence (ie. no background color)

**Comment by mrhoribu** [comment]
closes #1

**Pull request merged**